src/utility.js

```diff
@@ -0,0 +1,134 @@
+import Project from './project';
+
+const projectsList = document.getElementById('project');
+const editProjectsList = document.getElementById('editProject');
+
+function toggleShowElement(element) {
```

Comment:
- [ ] The `function` syntax should be replaced with ES6 syntax.
```javascript
const toggleShowElement = () => {}
```
- [ ] Do the same for all functions